samplers/jaegerremote/example/go.mod

```diff
@@ -0,0 +1,13 @@
+module go.opentelemetry.io/contrib/samplers/jaegerremote/example
+
+go 1.16
+
+require (
+	github.com/davecgh/go-spew v1.1.1
+	go.opentelemetry.io/contrib/samplers/jaegerremote v0.22.0
+	go.opentelemetry.io/otel v1.1.0
+	go.opentelemetry.io/otel/exporters/stdout/stdouttrace v1.1.0
+	go.opentelemetry.io/otel/sdk v1.1.0
+)
+
+replace go.opentelemetry.io/contrib/samplers/jaegerremote => ../
```

Comment:
should this line be removed before merge?

Reply:
I fixed this problem. Is it convenient for you to merge it? In addition, I look forward to the time when I can send the version.
[PR](https://github.com/kvrhdn/opentelemetry-go-contrib/pull/1)

Reply:
I haven't been authorized to merge request